docs/vault.md

```diff
@@ -98,4 +109,13 @@ The key rotation steps are the following:
 
 1. Rotate the master key for the secondary nodes one by one.
 2. Step down the primary and wait for another primary to be elected.
-3. Rotate the master key for the previous primary node.
\ No newline at end of file
+3. Rotate the master key for the previous primary node.
+
+### Prevent master key loss during creation
```

Comment:
I'd remove `during creation` completely. It follows from the test below and looks somewhat weird stylistically.

Reply:
Updated